src/ProjectOrigin.WalletSystem.IntegrationTests/REST/TransfersControllerTests.cs

```diff
@@ -178,14 +181,82 @@ public async Task TransferCertificate_Unauthorized()
         // Act
         var result = await controller.TransferCertificate(
             provider.GetRequiredService<ITestHarness>().Bus,
+            _unitOfWork,
             _fixture.Create<TransferRequest>());
 
         // Assert
         result.Result.Should().BeOfType<UnauthorizedResult>();
     }
 
     [Fact]
-    public async void TransferCertificate_PublishesCommand()
+    public async Task GetTransferStatus_Unauthorized()
```

Comment:
Same suggestion as above. The naming should tell why it is unauthorized

Reply:
removed